File: clases/clase1/ejercicios/ejer1_FizzBuzzKata/FizzBuzz.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ejer1_FizzbuzzKata
{
    public class FizzBuzz {

        private Dictionary<int, string> _dic;

        public FizzBuzz() {
            _dic = new Dictionary<int, string>();
            _dic.Add(3, "Fizz");
            _dic.Add(5, "Buzz");
        }

        public string Get(int i) {
            string result = "";
            foreach (var num in _dic.Keys) {
                if (i % num == 0) {
                    result += _dic[num];
                }
            }
            if (result != "")
                return result;
            else
                return i.ToString();
        }
    }
}

File: javascript/FizzBuzz/FizzBuzz.js
function FizzBuzz(){
	// associative array to emulate dictionary
	let dic = { '3': 'Fizz', '5': 'Buzz'};

	this.get = i => {
		let result = '';

		for(let key in dic){						
			if (i % parseInt(key) == 0)
			{				
				result += dic[key];
			}
		}

		if (result != '') 
			return result
		else
			return i.toString();
	};
}